test/e2e/app-dir/bun-externals/app/api/edge-bun-externals/route.ts

```diff
@@ -0,0 +1,26 @@
+import { NextResponse } from 'next/server'
+
+export const runtime = 'edge'
+
+// Edge runtime should not allow Bun imports
+// This test verifies that Bun modules are handled correctly in edge runtime
+export async function GET() {
+  const modules = [
+    { name: 'bunFfi', module: 'bun:ffi' },
+    { name: 'bunJsc', module: 'bun:jsc' },
+    { name: 'bunSqlite', module: 'bun:sqlite' },
+    { name: 'bunTest', module: 'bun:test' },
+    { name: 'bunWrap', module: 'bun:wrap' },
+    { name: 'bun', module: 'bun' },
+  ]
+
+  try {
+    for (const { module } of modules) {
+      await import(module)
+    }
+
+    return NextResponse.json('Did not throw')
+  } catch (e) {
+    return NextResponse.json('Threw')
```

Comment:
We should return the actual error(-message) and assert on that. This throw could be anything that may become incompatible later on.